lib/utils.js

```diff
@@ -194,3 +195,21 @@ function getRestifyClientOptions(opts) {
 
 	return (newOpts);
 }
+
+/*
+ * Convert the top level key names to lower case.
+ *
+ * @param {Object} with key names in mixed case
+ * @returns Object with all lower case keys
+ */
+function downcaseKeys(o) {
+	var keys = Object.keys(o);
+	keys.forEach(function (key) {
+		var l_key = key.toLowerCase();
+		if (key !== l_key) {
+			o[l_key] = o[key];
+			delete o[key];
```

Comment:
This will change the ordering of the object keys - I don't know if that matters so I'm just mentioning it.